src/DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php

```diff
@@ -220,7 +220,7 @@ public function process(ContainerBuilder $container): void
                     $groups[$resolvedGroupName]['keep_open'] = $groupDefaults[$resolvedGroupName]['keep_open'];
                 }
             }
-        } elseif ($container->getParameter('sonata.admin.configuration.sort_admins')) {
+        } elseif (true === $container->getParameter('sonata.admin.configuration.sort_admins')) {
```

Comment:
other solution is https://github.com/sonata-project/SonataAdminBundle/pull/7340/files#diff-305f77a1e9e7ffd017a5fc0f0b5a3ac360bf6f1e48c557e028d9d462bcc60104R165

It would be consistent with every other assert. (and the day the parameter type will be understood, the assert will be reported as an error)

Reply:
Done.